Issue 02 – ETL Assessments Data

ebranch13 opened the pull request:
# Description

Extracts, prepares, loads, and parallelizes the OPA assessments data.

_Describe your changes and why you're making them. Please include the context, motivation, and relevant dependencies._

Resolves #2 

## Type of change

- [x] New feature
- [ ] Bug fix
- [ ] Small fix/change
- [ ] Documentation

## What should the reviewer know?


## Post-merge follow-ups


- [x] No action required
- [ ] Actions required (specified below)

ebranch13 committed:
folder organization

solanoamy committed:
Duplicate issue01 folders to use as templates for issue02

ebranch13 committed:
delete .venv and .gitignore it!

solanoamy committed:
install python packages

solanoamy committed:
Setup requirements

ebranch13 committed:
Merge branch 'issue-02' of https://github.com/Weitzman-MUSA-GeoCloud/s25-team1-cama into issue-02

solanoamy committed:
locate key, fiddling with console

ebranch13 committed:
Merge branch 'issue-02' of https://github.com/Weitzman-MUSA-GeoCloud/s25-team1-cama into issue-02

solanoamy committed:
Tweaked code for deployment of 2.1 extract assessments, successful

ebranch13 committed:
Merge branch 'issue-02' of https://github.com/Weitzman-MUSA-GeoCloud/s25-team1-cama into issue-02

solanoamy committed:
edit 2.2 code to reflect assessments file

ebranch13 committed:
issue 2.2

solanoamy committed:
remove lines with geospatial function

ebranch13 committed:
issue 2.2

ebranch13 committed:
Update .gitignore

Ignore all data files (.csv and .jsonl)

solanoamy committed:
Remove duplicates from requirements file and update terminal code

requirements.txt had duplicate lines for different versions, so i removed those to reflect the most recent versions.

in issue2.2-terminal-code.sh, the file had been copied from issue01's folders and still reflected the function names from issue01, so i updated those.

solanoamy committed:
task02.3 complete

Updated SQL queries to reflect fields in assessments dataset and update filenames in main.py and terminal code files.

solanoamy committed:
renamed workflow file to reflect assessments and new URLs for assessments function

ebranch13 committed:
Revert "Update .gitignore"

This reverts commit b228306e063aafc605295331a92df34bacf692cd, reversing
changes made to da5a35c8ee78ddefa89376bc6aa98a04bf85ffb8.

ebranch13 committed:
Revert "delete .venv and .gitignore it!"

This reverts commit 92a9f1466cb1bb2bfd75798d17b63cec926b0303.

disandroa commented on tasks/2_extract_opa_assessments_data/2.2_prepare_opa_assessments_data/.Rhistory:
hmmm what is this .Rhistory file doing here.

disandroa commented on tasks/2_extract_opa_assessments_data/requirements.txt:
is this file necessary? in tasks 1 and 3 we don't have a requirements.txt file in the parent file, only in the x.1, x.2, x.3, and x.4 subfolders.

solanoamy committed:
Delete tasks/2_extract_opa_assessments_data/requirements.txt

Probably created from the multiple merge conflicts we encountered and not necessary for functionality.

solanoamy committed:
Delete tasks/2_extract_opa_assessments_data/2.2_prepare_opa_assessments_data/.Rhistory

Also not sure how this got here, and we had trouble moving it to the branch made to delete any venv folders created. Also had trouble in GitHub desktop, so hopefully this works here

disandroa reviewed (approved):
All changes look good! The code related to this issue successfully extracts, prepares, and loads in OPA assessments data.
